exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -133,21 +133,25 @@ def around_request
 
         def send_bytes(bytes, timeout:) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/MethodLength, Metrics/PerceivedComplexity
           return FAILURE if bytes.nil?
+          @metrics_reporter.record_value('otel.otlp_exporter.bundle_bytes', value: bytes.bytesize)
```

Comment:
I lean towards naming this attribute closer to how we name things using semantic conventions. I realize that this is not the _entire_ size of the request but perhaps something like this:

`otel.otlp_exporter.bundle.uncompressed_size`
`otel.otlp_exporter.bundle.compressed_size`

WDYT?

https://opentelemetry.io/docs/reference/specification/trace/semantic_conventions/rpc/#events